tests/shell/shell_recorder/script.dat

```diff
@@ -0,0 +1,13 @@
+Storage: dump
+Mountpoint: system/testmount/test
+File: /tmp/test.dump
+MountArgs:
+DiffType: File
+< mount $File $Mountpoint $Storage
```

Comment:
Is there also an interactive way for using the recorder?
Why is the `<` needed here? Why not syntactically distuingish the header and commands (e.g. via newline)?